marklogic-data-hub/src/main/java/com/marklogic/hub/FlowManager.java

```diff
@@ -60,7 +60,7 @@ public FlowManager(HubConfig hubConfig) {
         this.stagingClient = hubConfig.newStagingClient();
         this.finalClient = hubConfig.newFinalClient();
         this.jobClient = hubConfig.newJobDbClient();
-        this.jobManager = new JobManager(this.jobClient);
+        this.jobManager = new JobManager(this.jobClient, hubConfig.traceDbName);
```

Comment:
look at the `$config:JOB-DATABASE` variable in xquery. You don't need to pass this arround.

```xquery
import module namespace config = "http://marklogic.com/data-hub/config"
  at "/com.marklogic.hub/lib/config.xqy";
```